sdk/monitor/monitor-query-logs/review/monitor-query-logs-node.api.md

```diff
@@ -0,0 +1,236 @@
+    message: string;
+    resources?: string[];
+    target?: string;
+    value?: string;
+}
+
+// @public
+export interface ErrorInfo {
+    additionalProperties?: Record<string, any>;
+    code: string;
+    details?: ErrorDetail[];
+    innererror?: ErrorInfo;
+    message: string;
+}
+
+// @public
+export interface ErrorResponse {
+    error: ErrorInfo;
+}
+
+// @public
+export interface ExecuteOptionalParams extends QueryLogsOptions {
+}
+
+// @public
+export interface ExecuteWithResourceIdOptionalParams extends QueryLogsOptions {
+}
+
+// @public
+export enum KnownMonitorLogsQueryAudience {
+    AzureChina = "https://api.loganalytics.azure.cn",
+    AzureGovernment = "https://api.loganalytics.us",
+    AzurePublicCloud = "https://api.loganalytics.io"
+}
+
+// @public
+export interface LogsColumn extends Column {
+}
+
+// @public
+export interface LogsErrorInfo extends ErrorInfo {
+}
+
+// @public
+export interface LogsQueryBatchOptions extends OperationOptions {
+}
+
+// @public
+export type LogsQueryBatchResult = Array<LogsQueryPartialResult | LogsQuerySuccessfulResult | LogsQueryError>;
+
+// @public
+export class LogsQueryClient {
+    constructor(tokenCredential: TokenCredential, options?: LogsQueryClientOptions);
+    readonly pipeline: Pipeline;
+    queryBatch(batch: QueryBatch[], options?: LogsQueryBatchOptions): Promise<LogsQueryBatchResult>;
+    queryResource(resourceId: string, query: string, timespan: QueryTimeInterval, options?: LogsQueryOptions): Promise<LogsQueryResult>;
+    queryWorkspace(workspaceId: string, query: string, timespan: QueryTimeInterval, options?: LogsQueryOptions): Promise<LogsQueryResult>;
+}
+
+// @public
+export interface LogsQueryClientOptions extends ClientOptions {
+    apiVersion?: string;
+    audience?: string;
+}
+
+// @public
+export interface LogsQueryError extends Error {
+    code: string;
+    status: LogsQueryResultStatus.Failure;
+}
+
+// @public (undocumented)
+export interface LogsQueryOptions extends OperationOptions {
+    // (undocumented)
+    additionalWorkspaces?: string[];
+    // (undocumented)
+    includeQueryStatistics?: boolean;
+    // (undocumented)
+    includeVisualization?: boolean;
+    // (undocumented)
+    serverTimeoutInSeconds?: number;
+}
+
+// @public
+export interface LogsQueryPartialResult {
+    partialError: LogsErrorInfo;
+    partialTables: LogsTable[];
+    statistics?: Record<string, unknown>;
+    status: LogsQueryResultStatus.PartialFailure;
+    visualization?: Record<string, unknown>;
+}
+
+// @public
+export type LogsQueryResult = LogsQuerySuccessfulResult | LogsQueryPartialResult;
+
+// @public
+export enum LogsQueryResultStatus {
+    Failure = "Failure",
+    PartialFailure = "PartialFailure",
+    Success = "Success"
+}
+
+// @public
+export interface LogsQuerySuccessfulResult {
+    statistics?: Record<string, unknown>;
+    status: LogsQueryResultStatus.Success;
+    tables: LogsTable[];
+    visualization?: Record<string, unknown>;
```

Comment:
same comment about unknown.

Reply:
same as above https://github.com/Azure/azure-sdk-for-js/pull/35214#discussion_r2227354353